collection/collector.go

```diff
@@ -8,6 +8,11 @@ import (
 	pb "github.com/odpf/raccoon/proto"
 )
 
+type Event struct {
+	Type       string
+	EventBytes []byte
```

Comment:
will `data ` or `content` be a better name instead of `EventBytes`

Reply:
This interface is stale, so will remove this. Anyway this came from [here](https://github.com/odpf/proton/blob/main/odpf/raccoon/v1beta1/raccoon.proto#L68)